travvy-server/src/controllers/AuthenticationController.js

```diff
@@ -48,6 +48,17 @@ module.exports = {
         //retun the user that was found
         //if there is not a user that exists it will return null
         return user
+    },
+    //this method returns a user
```

Comment:
This function can be later utilized again for any feature that requires the system to grab the profile for a given user! Great